fundamentals/intro.tex

```diff
@@ -31,11 +31,10 @@ \section{Introduction}
 Dynamic nuclear fuel cycle analysis more realistically supports a range of
 simulation goals than static analysis \cite{piet_dynamic_2011}. Historically,
 dynamic nuclear fuel cycle simulators have calculated fuel cycle mass balances
-and performance metrics derived from them using software ranging from ad hoc
+and performance metrics derived from them using software ranging from 
 spreadsheet-driven flow calculators to highly specialized system dynamics
-modeling platforms. To date, current tools are often rigid rather than flexible,
-fleet-based rather than discrete, or privately distributed rather than open
-source. Each of these choices presents a challenge to modeling fidelity,
+modeling platforms. To date, current tools are often rigid,
+fleet-based, or privately distributed. Each of these choices presents a challenge to modeling fidelity,
```

Comment:
Hi @FlanFlanagan , we've gone back and forth about this sentence. The original version looked like your current version, but we added the "X rather than Y" instances because it was more clear what the choices were, exactly. Can you change this sentence back?